libs/common/src/assets/i18n/de.json

```diff
@@ -0,0 +1,892 @@
+{
+	"account.account_name_invalid": "Ungültiger Kontoname",
+	"account.accumulated_consumption": "Kumulierter Tagesverbrauch nach Funktion",
+	"account.active_config": "Aktive Konfiguration",
+	"account.add_stash": "Wissensbox hinzufügen",
+	"account.add_user": "Nutzer hinzufügen",
+	"account.admin.make_admin": "Administrator erstellen",
+	"account.admin.warning": "Wenn Sie diesem Benutzer die Administratorrolle zuweisen, kann er auf alle Optionen zugreifen und diese bearbeiten.",
+	"account.billing": "Zahlung & Abonnement",
+	"account.chart_error_media_seconds": "Die Daten zu „Medienstunden“ konnten nicht abgerufen werden",
+	"account.chart_error_processing_status": "Die Daten „zur Verarbeitung ausstehende Ressourcen“ konnten nicht abgerufen werden",
+	"account.chart_error_searches": "„Suchdaten“ können nicht abgerufen werden",
+	"account.chart_error_train_seconds": "Die Daten zu „Zugstunden“ können nicht abgerufen werden",
+	"account.chart_media_seconds": "Medienzeiten",
```

Comment:
This was translated to "media times" instead of "media seconds".

Reply:
English is: `Media hours` (I guess we changed the way to talk about those charts at some point, because I agree the translation keys doesn't make much sense here)